assets/RNAseq_report.Rmd

```diff
@@ -514,6 +1290,280 @@ cat("Plots for the [genes of interest](./differential_gene_expression/metadata/g
 
 <!-- In case KEGG analysis was also included; differentiating the cases with and without contrasts -->
 
+
+<!-- PA start -->
+```{r load_PA_libs, eval=params$pathway_analysis, echo=FALSE, message=FALSE, warning=FALSE, results = 'hide'}
+# Load libs invisibly
+invisible(lapply(c(
+"gprofiler2",
+"reshape2",
+"pathview",
+"AnnotationDbi",
+"optparse",
+"org.Mm.eg.db",
+"org.Hs.eg.db"
+), library, character.only=T))
+# Need to load library for your species! Or provide as parameter to biocmanager
```

Comment:
I thought this is solved by your `install(params$species_libraries)` code below already.